internal/cli/url_scheme.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type UrlScheme string
+
+const (
+	httpsScheme UrlScheme = "https"
+	httpScheme  UrlScheme = "http"
+)
+
```

Comment:
Mind adding some documentation on how to use this function and the expected format is for the url string? I think that additional insight would be valuable to have - even a snip it of such examples as a comment.